lib/searchjoy/track.rb

```diff
@@ -5,11 +5,23 @@ def search_with_track(term, options = {}, &block)
         block.call(body) if block
       end
 
-      if options[:track]
+      if options[:track] && options[:execute]
```

Comment:
Mh, good point. I guess hooking into `execute`will be sufficient. I´ll update the PR as soon as I can.